src/storage/src/table/batch_table/storage_table.rs

```diff
@@ -652,9 +653,15 @@ struct StorageTableInnerIterInner<S: StateStore, SD: ValueRowSerde> {
 
     output_indices: Vec<usize>,
 
+    /// If all values of `output_indices` is unique, set `exist_index_duplicate` false.
+    exist_index_duplicate: bool,
```

Comment:
Is it possible to encapsulate them into `ColumnMapping`? In which way the projection can also be less error-prone.